Extensions/keyboardShortcut.js

```diff
@@ -1,236 +1,102 @@
-	 * - CTRL            - G               - SELECT          - F7
-	 * - ALT             - H               - NUMPAD_0        - F8
-	 * - PAUSE/BREAK     - I               - NUMPAD_1        - F9
-	 * - CAPS            - J               - NUMPAD_2        - F10
-	 * - ESCAPE          - K               - NUMPAD_3        - F11
-	 * - SPACE           - L               - NUMPAD_4        - F12
-	 * - PAGE_UP         - M               - NUMPAD_5        - NUM_LOCK
-	 * - PAGE_DOWN       - N               - NUMPAD_6        - SCROLL_LOCK
-	 * - END             - O               - NUMPAD_7        - ;
-	 * - HOME            - P               - NUMPAD_8        - =
-	 * - ARROW_LEFT      - Q               - NUMPAD_9        - ,
-	 * - ARROW_UP        - R               - MULTIPLY        - -
-	 * - ARROW_RIGHT     - S               - ADD             - /
-	 * - ARROW_DOWN      - T               - SUBTRACT        - `
-	 * - INSERT          - U               - DECIMAL_POINT   - [
-	 * - DELETE          - V               - DIVIDE          - \
-	 * - A               - W               - F1              - ]
-	 * - B               - X               - F2              - "
-	 *
-	 * Use one of keyName as a string. If key that you want isn't in that list,
-	 * you can also put its keycode number in keyName as a number.
-	 *
-	 * callback is name of function you want your shortcut to bind to. It also
-	 * returns one KeyboardEvent parameter.
-	 *
-	 * Following are my default keybinds, use them as examples.
+	 * Binds a keyboard shortcut using Mousetrap.
+	 * @param {string} key - The Mousetrap keybind.
+	 * @param {boolean | undefined} staticCondition - A static condition.
+	 * @param {(event: KeyboardEvent) => void} callback - Callback function for the event.
 	 */
-
-	// Ctrl + Tab and Ctrl + Shift + Tab to switch sidebar items
-	registerBind("TAB", true, false, false, rotateSidebarDown);
-	registerBind("TAB", true, true, false, rotateSidebarUp);
-
-	// Ctrl + Q to open Queue page
-	registerBind("Q", true, false, false, clickQueueButton);
-
-	// Shift + H and Shift + L to go back and forward page
-	registerBind("H", false, true, false, clickNavigatingBackButton);
-	registerBind("L", false, true, false, clickNavigatingForwardButton);
-
-	// PageUp, PageDown to focus on iframe app before scrolling
-	registerBind("PAGE_UP", false, true, false, focusOnApp);
-	registerBind("PAGE_DOWN", false, true, false, focusOnApp);
-
-	// J and K to vertically scroll app
-	registerBind("J", false, false, false, appScrollDown);
-	registerBind("K", false, false, false, appScrollUp);
-
-	// G and Shift + G to scroll to top and to bottom
-	registerBind("G", false, false, false, appScrollTop);
-	registerBind("G", false, true, false, appScrollBottom);
-
-	// M to Like/Unlike track
-	registerBind("M", false, false, false, Spicetify.Player.toggleHeart);
-
-	// Forward Slash to open search page
-	registerBind("/", false, false, false, openSearchPage);
```

Comment:
Why did this one not get ported?

Reply:
as per the pr message spotify has these built in now just with different syntax and an actual menu (ctrl +?) to see binds, its pointless having dupes....

Reply:
Fair on this one! Although the like in the new Keyboard shortcuts seems a bit too complicated and we could probably keep the "m" one, what do you think?

Reply:
I dont like the idea of adding keybinds just for the sake of a simpler key combination, however if you want it i can do it just lmk